VMEncryption/main/handle.py

```diff
@@ -660,10 +663,20 @@ def enable():
     try:
         hutil.do_parse_context('Enable')
         logger.log('Enabling extension')
-
         public_settings = get_public_settings()
         logger.log('Public settings:\n{0}'.format(json.dumps(public_settings, sort_keys=True, indent=4)))
-        cutil = CheckUtil(logger)
+        checkUtil = CheckUtil(logger)
+        imdsStoredResults=IMDSStoredResults(logger=logger,encryption_environment=encryption_environment)
+        iMDSUtil = IMDSUtil(logger)
+        try:
+            checkUtil.preInitializationCheck(logger,imdsStoredResults=imdsStoredResults,iMDSUtil=iMDSUtil)
+        except Exception as ex:
+             hutil.do_exit(exit_code=CommonVariables.unknown_error,
```

Comment:
use configuration_error otherwise it would e considered an unexpected failure.

Reply:
done